bin/ci/cucumber.rb

```diff
@@ -2,105 +2,58 @@
 
 require "run_loop"
 require "luffa"
+require "pry"
```

Comment:
Please refactor this script:
1) remove noise changes, like spaces, `require "pry"` and etc. 
2) I think we don't need `which_languages` function at all.
You can use something like:
```
languages = ["en", "ru"]
languages.each do |lang|
  args.push("APP_LANG=#{lang}" APP_LOCALE=#{lang}")
```
3) 
```
device = RunLoop::Core.default_simulator(xcode)
device = device.gsub! ' ', ''
device = device.gsub! '(', ''
device = device.gsub! ')', ''
``` 
what we do in this code? Can we refactor it a bit to make more readable?

4)
I see that you determine device two times:
`device` variable,
`match` variable,
Why we need to do that twice?
5) Please remove unused variables